ETC.tsv

```diff
@@ -4608,9 +4608,9 @@ ETC_20150317_004785	Ask about the Divine Day
 ETC_20150317_004786	Ask what happened next
 ETC_20150317_004787	I found the lever handle!
 ETC_20150317_004788	I found the information about the flow of horsepower.
-ETC_20150317_004789	Special information can not be found.
+ETC_20150317_004789	Special information could not be found.
 ETC_20150317_004790	Please remove the trap which attracts the monster!
-ETC_20150317_004791	More approaches and it is dangerous.
+ETC_20150317_004791	It grows more dangerous as you get closer
```

Comment:
The original translation is not implying that you are getting closer to the danger, its implying the danger is approaching you. Maybe try "More monsters are approaching, it is dangerous."